cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html

```diff
@@ -11,23 +11,27 @@
 
    See [GHE]/flapjack/Modules-V1/wiki/50-50-Image-&-Text
 
-   settings:              An object with the following options for settings.
-   settings.title:        A string containing title of module
-   settings.image_path:   A string containing the url for the image
-   settings.image_alt:    A string containing alt text for screenreaders
+   settings:               An object with the following options for settings.
```

Comment:
I'd be on board with that. For readability, I kind of prefer the indentation vs writing out `settings.` repeatedly, but looking across the codebase that's how it's done, so updated here to align with that  in https://github.com/cfpb/cfgov-refresh/commit/51c846244e362a15ee086f8bd11c6d3045cad1e5 (also fixes slight indentation issue in link blob).
Opened https://github.com/cfpb/cfgov-refresh/issues/1083 for it.